hw/ip/otp_ctrl/data/otp_ctrl.hjson

```diff
@@ -1515,6 +1515,62 @@
     }
   ]
 
+  features: [
+    {
+      name: "OTP_CTRL.VENDOR_TEST_PARTITION"
+      desc: '''Vendor test partition is intended to be used for OTP programming smoke check during manufacturing flow.
+      In this partition, uncorrectable errors will not lead to fatal errors and alerts.
```

Comment:
That second line here is not fully correct:
```
ECC uncorrectable errors seen on the functional prim_otp interface will not lead to an alert for this partition. Instead, such errors will be reported as correctable ECC errors.
```
https://opentitan.org/book/hw/ip/otp_ctrl/doc/theory_of_operation.html#partition-listing-and-description

This is done so that you can mess with the bits and the OTP controller will not shut down by accident.

Reply:
fixed